src/renderer_vk.cpp

```diff
@@ -3701,11 +3904,13 @@ VK_IMPORT_DEVICE
 			viewportState.scissorCount  = 1;
 			viewportState.pScissors     = NULL;
 
+			VkSampleCountFlagBits rasterizerMsaa = (isValid(m_fbh) && !!(BGFX_STATE_MSAA & _state) ? m_textures[m_frameBuffers[m_fbh.idx].m_attachment[0].handle.idx].m_sampler.Sample : VK_SAMPLE_COUNT_1_BIT);
```

Comment:
Also here I'm not quite sure if it's the best way to know which is the current framebuffer color attachment MSAA sampler. This was the best way I found out, but if you have a better idea, feel free to point out a better solution :)